webapp/views/App/views/Home/SurveyInfo/LanguagesEditor/LanguagesEditor.js

```diff
@@ -30,6 +30,7 @@ const LanguagesEditor = (props) => {
         }}
         requiredItems={1}
         readOnly={readOnly}
+        itemKey="key"
```

Comment:
itemKey already defaults to "key", right?

Reply:
i think that not but i will check

Reply:
yes into the component default props i will clean that then, thanks!